discussion/app/views/fragments/commentShare.scala.html

```diff
@@ -0,0 +1,38 @@
+@(comment: Comment)
+@import org.jsoup.Jsoup
+@import org.jsoup.safety.Whitelist
+@import org.apache.commons.lang.StringEscapeUtils
+
+<div class="sharing-component d-comment__action">
+
+    <div class="sharing-text">@fragments.inlineSvg("share", "icon", List("comment-share-icon")) Share</div>
+
+    <div class="sharing-buttons" data-link-name="comment social">
+    @defining(s"https://gu.com${comment.discussion.key}#comment-${comment.id}") { permalink =>
+        @defining(StringEscapeUtils.unescapeHtml(Jsoup.clean(comment.body, Whitelist.none()))) { commentBody =>
+            <a href="@ShareLinks.createShareLinkForComment(platform = Facebook, href = permalink, text = "", quote = Some(s"""${comment.profile.displayName} commented: "$commentBody"""")).href"
+            target="_blank"
+            class="social__action social-icon-wrapper"
+            data-link-name="social-comment : facebook">
+                <span class="inline-icon__fallback button share-modal__item share-modal__item--facebook">
```

Comment:
The `share-modal` classes suggest to me that we're re-using styles here that are designed for modals, can you pull them out and make them more generic?